plugins/lightspeed-backend/src/service/types.ts

```diff
@@ -2,10 +2,10 @@ import {
   // DiscoveryService,
   // HttpAuthService,
   // UserInfoService,
-  LoggerService,
+  type LoggerService,
```

Comment:
```suggestion
  LoggerService,
```
Move `type` to the top level and remove the comments.